conf/messages

```diff
@@ -1444,6 +1444,26 @@ incomeSources.ceaseForeignProperty.checkDetails.dateStopped           = Date bus
 incomeSources.ceaseForeignProperty.checkDetails.confirm               = Confirm and continue
 incomeSources.ceaseForeignProperty.checkDetails.change                = Change
 
+## Cease - Check Answers ##
+cease-check-answers.title                                     = Check your answers
+cease-check-answers.heading                                   = Sole trader
```

Comment:
❓ maybe better to call this caption, rather than heading as to be more clear on terms